### PR TITLE
[stable/newrelic-infrastructure] Retain support for Kubernetes 1.8

### PR DESCRIPTION
**Pull request by coreypobrien**
#4917 switched the Daemonset to `apps/v1`. This change re-adds support for Kubernetes 1.8 using `apps/v1beta2`.

### CODE REVIEW

File: stable/newrelic-infrastructure/templates/daemonset.yaml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {{- if and .Values.licenseKey .Values.cluster }}
+{{- if semverCompare "< 1.9-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: apps/v1beta2
```
Comment (unguiculus) [inline stable/newrelic-infrastructure/templates/daemonset.yaml]: I'd just go for this and remove the if block. This is what we commonly use now.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
cc @rk295

**Review by unguiculus — APPROVED** [review]

**Comment by coreypobrien** [comment]
@unguiculus Anything else needed here?

**Comment by unguiculus** [comment]
Sorry for the delay.

**Pull request merged**